WordPress/WordPressTest/PostTests.swift

```diff
@@ -0,0 +1,168 @@
+import Foundation
+import XCTest
+
+@testable import WordPress
+
+class PostTests: XCTestCase {
+
+    private let context: NSManagedObjectContext = {
+        let context = NSManagedObjectContext(concurrencyType: .MainQueueConcurrencyType)
+        context.parentContext = TestContextManager.sharedInstance().mainContext
+
+        return context
+    }()
+
+    private func newTestBlog() -> Blog {
+        return NSEntityDescription.insertNewObjectForEntityForName(BlogEntityName, inManagedObjectContext: context) as! Blog
+    }
+
+    private func newTestPost() -> Post {
+        return NSEntityDescription.insertNewObjectForEntityForName(PostEntityName, inManagedObjectContext: context) as! Post
+    }
+
+    private func newTestPostCategory() -> PostCategory {
+        return NSEntityDescription.insertNewObjectForEntityForName(PostCategoryEntityName, inManagedObjectContext: context) as! PostCategory
+    }
+
+    private func newTestPostCategory(name: String) -> PostCategory {
+        let category = newTestPostCategory()
+        category.categoryName = name
+
+        return category
+    }
+
+    override func setUp() {
+        super.setUp()
+    }
+
+    override func tearDown() {
+        context.rollback()
+        super.tearDown()
+    }
+
+    func testThatNoCategoriesReturnEmptyStringWhenCallingCategoriesText() {
+        let post = newTestPost()
+        let categoriesText = post.categoriesText()
+
+        XCTAssert(categoriesText == "")
+    }
+
+    func testThatSomeCategoriesReturnAListWhenCallingCategoriesText() {
+
+        let post = newTestPost()
+
+        post.categories = Set<PostCategory>(arrayLiteral: newTestPostCategory("1"), newTestPostCategory("2"), newTestPostCategory("3"))
```

Comment:
Just as a side note, here's one example of what I meant:

<img width="745" alt="screen shot 2016-05-27 at 9 07 17 am" src="https://cloud.githubusercontent.com/assets/1836005/15607598/77fe483c-23ea-11e6-9bbb-4120602899bf.png">